src/main/java/it/gov/pagopa/payment/connector/rest/merchant/MerchantConnector.java

```diff
@@ -0,0 +1,10 @@
+package it.gov.pagopa.payment.connector.rest.merchant;
+
+import it.gov.pagopa.payment.connector.rest.merchant.dto.MerchantDetailDTO;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+
+public interface MerchantConnector {
+    MerchantDetailDTO merchantDetail(@RequestHeader("x-merchant-id") String merchantId,
+                                     @PathVariable("initiativeId") String initiativeId);
```

Comment:
these annotations are useless here
```suggestion
    MerchantDetailDTO merchantDetail(String merchantId, String initiativeId);
```